src/test/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpointIT.java

```diff
@@ -2705,7 +2708,11 @@ private void shouldReturn404WhenPostCreateCaseWithNoCreateFieldAccess(String use
     }
```

Comment:
these test classes are becoming too big. 
Shall we introduce a integration test class dedicated to Case Roles?